src/mpyl/steps/deploy/dagster.py

```diff
@@ -0,0 +1,138 @@
+"""
+Step to deploy a dagster user code repository to k8s
+"""
+from logging import Logger
+
+import yaml
+
+from .k8s import (
+    helm,
+    get_config_map,
+    rollout_restart_deployment,
+    cluster_config,
+    replace_config_map,
+    update_config_map_field,
+    get_version_of_deployment,
+)
+from .k8s.resources.dagster import to_user_code_values, to_grpc_server_entry
+from .. import Step, Meta, ArtifactType, Input, Output
+from ...project import Stage, Target, get_env_variables
+from ...utilities.docker import DockerConfig
+
+
+class DeployDagster(Step):
+    def __init__(self, logger: Logger) -> None:
+        super().__init__(
+            logger,
+            Meta(
+                name="Dagster Deploy",
+                description="Deploy a dagster user code repository to k8s",
+                version="0.0.1",
+                stage=Stage.DEPLOY,
+            ),
+            produced_artifact=ArtifactType.NONE,
+            required_artifact=ArtifactType.DOCKER_IMAGE,
+        )
+
+    def __convert_to_helm_release_name(self, name: str, tag: str) -> str:
```

Comment:
This method could be in a helm utilities module

Reply:
done in
[34b4331](https://github.com/Vandebron/mpyl/pull/146/commits/34b4331b34b4fa1ada15dd88ce7115eb7e08464c)